.github/workflows/weekly.yml

```diff
@@ -0,0 +1,32 @@
+name: weekly
+
+on:
+  schedule:
+    # this script will run each Monday at 14:14
+    - cron: "14 14 * * 1"
+  # Uncomment below to test.
+  # push:
+  #   branches: [gha-test-*, canary, auto]
+
+jobs:
+  dep-report:
+    runs-on: ubuntu-latest
+    container:
+      image: diem/build_environment:main
```

Comment:
We could make this a matrix build and report on main/releases-1.2

Reply:
We're only currently reporting on dep changes in main so I don't think there's much value right now in looking at the release branch.

Reply:
what is a matrix build?
Are you suggesting to replace main with a target_branch variable which would then run on both main and release-1.2?

Reply:
As I mentioned, lets not do this because we already don't worry about deps on the release branch. We can look at changing this in the future if we end up caring about the release branch